apps/cancel-request/fields/index.js

```diff
@@ -8,111 +8,86 @@ module.exports = {
   },
   'cnc-main-applicant-full-name': {
     mixin: 'input-text',
-    validate: ['required', { type: 'maxlength', arguments: 250 }, { type: 'minlength', arguments: 3 }],
-    legend: {
-      className: 'bold'
-    }
+    validate: ['required', { type: 'maxlength', arguments: 250 }, { type: 'minlength', arguments: 3 }, 'notUrl'],
+    className: ['govuk-input', 'govuk-!-width-two-thirds']
```

Comment:
can remove this className array to match figma, and its already govuk-input from hof